managing-you-data-lifecycle-with-time-to-live-tables.md

```diff
@@ -0,0 +1,119 @@
+### Jaeger Tracing
+
+[Jaeger Tracing](https://github.com/jaegertracing/jaeger) is a distributed tracing platform that supports the OpenTracing standard and provides experimental support for OpenTelemetry. Jaeger can use Cassandra and Elasticsearch as the storage backends, and also provides a gRPC plugin for integration with other storage systems.
+
+We use the plugin to run Jaeger on TiDB. **TiDB brings Jaeger the benefit of TiKV's high throughput and scalability. It automatically manages the lifecycle of historical tracing data, lowering system operations cost.**
+
+### Kubernetes events
+
+The Kubernetes `event` object records all events happening in the cluster. These events help you troubleshoot exceptional behaviors and audit cluster operations. When the Kubernetes cluster is large and sees frequent changes, you might choose to store the cluster events in a separate etcd cluster. However, when Kubernetes is frequently updated, the cluster generates a large amount of events. Because of limited storage space, the etcd cluster can't record all Kubernetes events, but only the recent ones.
+
+To address this issue, **you can store Kubernetes events in TiDB, which stores more data in a longer period of time**. In addition, **TiDB's secondary index enables you to quickly and flexibly query the stored events**. With the help of the TTL table, the historical events stored in TiDB automatically expire, and your storage space is saved.
+
+### MQTT QoS
+
+Message Queuing Telemetry Transport (MQTT) is a lightweight publish/subscribe protocol for Internet of Things (IoT) messaging. Because mobile and IoT devices may have unstable network connections, the system needs to persist messages so that even when the client goes offline for a long time, the message delivery still reaches quality of service (QoS) Level 1 or 2.
+
+**A storage backend with a built-in data expiration mechanism is well suited for applications that require reliable and timely delivery**, such as breaking news, AMBER Alerts, and weather forecasts.
+
+During the Hackathon, we adapted a popular MQTT broker to make it support MySQL as backend storage. Because TiDB is MySQL compatible, **TiDB's TTL table can automatically delete the expired data according to the retention cycle set by the user**.
+
+## Summary
+
+In an era of everything connected with everything, the volume of data is rapidly growing. How to support more data with fewer costs is a question that all DBMSs try to answer. Our answer to this question is the TTL table.
+
+For TiDB, the heart of this question is how to balance the cost of TiDB and the value of data it carries. The more relevant and fresher data is, the more value TiDB creates for your business. After all, time waits for no one, and it's better to make room for more valuable data.
+
+_At TiDB Hackathon 2020, many excellent, interesting projects were born. We'll be telling you about them in future [blog posts](https://pingcap.com/blog/tag/Hackathon). Stay tuned._
```

Comment:
Here, before "TiDB Hackathon 2020," you use "at", but you used "in" in the preceding content. You can update "in TiDB Hackathon 2020" to "at TiDB Hackathon 2020" to ensure consistent language style.